IntegrationTests/ReducedCombinedConfig/script/makeProject.tcl

```diff
@@ -0,0 +1,65 @@
+# Create Vivado project, with user HDL files & IP.
+# Run this in IntegrationTests/xyz/script/ 
+
+# Create project
+set projName "Work"
+set FPGA "xcvu13p-fhgb2104-2-e"
```

Comment:
@aryd Does this not cause problems with the part for the HLS modules being a VU7P?:
https://github.com/cms-L1TK/firmware-hls/blob/a3e5decfe1ed903ab9c66c948ba16e3811c6fc68/project/settings_hls.tcl#L22

Reply:
Had not noticed any problems; but we should probably have this consistent - does it need to be set in more than one place?

Reply:
Consistency would be good. For the HLS modules, it needs to be set in project/settings_hls.tcl, and for each of the chains, it needs to be set in makeProject.tcl in that chain's script/ directory.

Here, I would suggest setting `FPGA` to the VU7P for now, to match the other chains and since this chain should easily fit on one. Then we can think about how to make the parts consistent as we transition our testing to the VU13P.